Workbooks/LogAnalytics Workspace/AtScale/Workspace Insights At Scale.workbook

```diff
@@ -0,0 +1,465 @@
+            "type": 1,
+            "description": "Used for 'No Subscriptions' workbook template",
+            "isRequired": true,
+            "isHiddenWhenLocked": true,
+            "criteriaData": [
+              {
+                "criteriaContext": {
+                  "operator": "Default",
+                  "rightValType": "param",
+                  "resultValType": "static",
+                  "resultVal": "Log Analytics Workspace"
+                }
+              }
+            ],
+            "value": "Log Analytics Workspace"
+          },
+          {
+            "id": "1c84117c-7449-4484-b693-00f41a578bd6",
+            "version": "KqlParameterItem/1.0",
+            "name": "ResourceImageUrl",
+            "type": 1,
+            "description": "Used for 'No Subscriptions' workbook template",
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 0
+            },
+            "timeContextFromParameter": "TimeRange"
+          }
+        ],
+        "style": "above",
+        "queryType": 1,
+        "resourceType": "microsoft.resourcegraph/resources"
+      },
+      "name": "Log Analytics Workspace parameters",
+      "styleSettings": {
+        "margin": "15px 0 0 0"
+      }
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "template",
+        "loadFromTemplateId": "community-Workbooks/Common/noSubscriptions",
+        "items": []
+      },
+      "conditionalVisibility": {
+        "parameterName": "Subscription",
+        "comparison": "isEqualTo",
+        "value": ""
+      },
+      "name": "No Subscriptions group"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "{Message}\r\n_Log Analytics workspace(s)_\r\n<br />\r\n<br />"
+      },
+      "name": "resource summary",
+      "styleSettings": {
+        "margin": "15px 0 10px 10px"
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "where type =~ 'microsoft.operationalinsights/workspaces'\r\n| where '{Workspace:name}' has name\r\n| extend \r\n  state = trim(' ', tostring(properties.provisioningState)),\r\n  sku = trim(' ', tostring(properties.sku.name)),\r\n  skuUpdate = trim(' ', tostring(properties.sku.lastSkuUpdate)),\r\n  retentionDays = trim(' ', tostring(properties.retentionInDays)),\r\n  dailyquotaGB  = trim(' ', tostring(properties.workspaceCapping.dailyQuotaGb))\r\n| extend dailyquotaGB = iif(dailyquotaGB !=-1.0, dailyquotaGB,\"Not set\")\r\n| extend skuUpdate = iif(strlen(skuUpdate) > 0, skuUpdate,\"Unknown\")\r\n//| extend sentinel = iif(toint(retentionDays) < 90,\"If you have Sentinel, you can change your retention to 90days (free)?\",\"\")\r\n| project ['Workspace']=id, Name=name, ['Resource Group']=resourceGroup, location, ['Data Retention (days)']=retentionDays, \r\n  ['Last known SKU update']=skuUpdate, ['Daily Data Cap']=dailyquotaGB, ['Licence']=sku",
```

Comment:
convert resourceGroup to an id so that it can be linkable `['Resource Group']=strcat("/subscriptions/", subscriptionId, "/resourceGroups/", resourceGroup)`

Also noticed that `//| extend sentinel = iif(toint(retentionDays) < 90,\"If you have Sentinel, you can change your retention to 90days (free)?\",\"\")\r\n` is commented out, can we remove that?